lib/blocs/workspaces_cubit/workspaces_cubit.dart

```diff
@@ -0,0 +1,38 @@
+
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:twake/blocs/workspaces_cubit/workspace_state.dart';
+import 'package:twake/models/account/account.dart';
+import 'package:twake/models/globals/globals.dart';
+import 'package:twake/models/workspace/workspace.dart';
+import 'package:twake/repositories/workspaces_repository.dart';
+
+class WorkSpacesCubit extends Cubit<WorkspaceState> {
```

Comment:
Totally overlooked it last time, Workspace (without camelCasing for naming consistency)

Reply:
done